src/main/scala/viper/gobra/reporting/VerifierResult.scala

```diff
@@ -13,7 +13,7 @@ sealed trait VerifierResult
 object VerifierResult {
 
   case object Success extends VerifierResult
-  case class Failure(errors: Vector[VerifierError]) extends VerifierResult
+  case class Failure(errors: Either[Vector[VerifierError], Vector[VerifierError]]) extends VerifierResult
```

Comment:
It is not clear here in which case we should put our errors on `Left` and when we should put them on `Right`